.gitignore

```diff
@@ -8,13 +8,16 @@ node_modules
 
 # Test output:
 .nyc_output
-coverage/
+coverage
```

Comment:
These slashes were superfluous/confusing, so I removed them.

Reply:
I have a habit of using `/ on directories. I find it helpful to make it clear which are directories vs. which are files.